docs/source/fine-performance-metrics.rst

```diff
@@ -0,0 +1,198 @@
+Fine Performance Metrics
+========================
+.. note::
+
+    This is an experimental feature and may rapidly change without a deprecation cycle.
+
+You may want to investigate where your Dask workload spends the majority of its time;
+not only on which tasks, but also *doing what* while running said tasks.
+*Fine performance metrics* are collected automatically by Dask and attempt to answer
```

Comment:
Could we first explain what they do before saying what you can use them for?

Reply:
tweaked